.github/workflows/main.yml

```
@@ -67,6 +69,8 @@ jobs:
         with:
           name: mrc_rococo_2000_specs-${{ github.sha }}
           path: mrc_specs.tar
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

@saraswatpuneet is this needed?
```suggestion
      - name: Set up QEMU
        if: matrix.target == 'build-node'
        uses: docker/setup-qemu-action@v2
```